content/guides/building/get-source-git.md

```diff
@@ -40,141 +44,150 @@ git clone https://review.haiku-os.org/buildtools
-<p>If the e-mail address used in your commits does not match the one in your github account, you will need to add and verify it in <a href="https://review.haiku-os.org/settings/#EmailAddresses">Gerrit E-Mail address settings</a>.</p>
+<a name="dev_access"></a>
+<h3>Git Access - Contributors and patch submitters</h3>
 
-<h4>Build Tools:</h4>
+So, you want to contribute to Haiku. That's awesome, thank you for your
+interest! This section will include instructions for setting up your local Git
+environment, an account on our Gerrit instance, as well as some guidelines on
+how to work with other contributors to get your changes included. This guide
+assumes that you are new to contributing to open-source projects. If that
+is not the case, you can skip some of the sections.
 
-The `<login>@` is only needed if your currently logged in username doesn't match your `review.haiku-os.org` username.
+<h4>A visual overview of our Git workflow</h4>
+<img src='/files/gitProcess_0.png' alt='A diagram describing the way our Git workflow works. This picture is there for aesthetic purposes and can be skipped. After a repository is cloned locally, if you have an idea, you can convert that idea into a commit using food (a pizza is depicted in the diagram). That commit will receive some change proposals, which will result in some further changes. The circle continues until the change is ready to be included in Haiku itself.'>
 
-```sh
-git clone "ssh://<login>@git.haiku-os.org/buildtools" && scp -p <login>@git.haiku-os.org:hooks/commit-msg "buildtools/.git/hooks/"
-```
+<h4>Configuring Git locally<a name="configure_env"></a></h4>
 
-<h4>Haiku:</h4>
+<div class="alert alert-danger">
+<strong>This section is important!</strong> Failing to configure Git properly before a commit will result in an incorrect name being used in your commit, making it impossible to give you well-deserved credit for your work.</div>
 
-The `<login>@` is only needed if your currently logged in username doesn't match your `review.haiku-os.org` username.
+A global Git configuration is stored in Git configuration directory (`~/.git/` or for Haiku: `~/config/settings/git/`) and will be included in <strong>every</strong> commit.
 
 ```sh
-git clone "ssh://<login>@git.haiku-os.org/haiku"
-scp -p <login>@git.haiku-os.org:hooks/commit-msg "haiku/.git/hooks/"
+git config --global user.name "John Doe"
+git config --global user.email "john.doe@example.com"
 ```
 
-<h4>Switching from read-only to write access</h4>
-
-Want to submit your first patch? Thanks, that's great! You don't need to checkout the sources again. Instead you can update your existing copy of the source to use the commiter access. Just change the remote URL:
+Although most contributors tend to disclose their real names, we do not have
+a real name policy and plenty of contributors contribute to Haiku using a pseudonym.
+That means that it is possible for anyone to contribute.
```

Comment:
While this is an important topic to you, I don't think this is the right place to mention it in this way.

```suggestion
You are not required to use your "real" name, a pseudonym is fine and several Haiku contributors chose to do that. However, make sure the e-mail address is correct and working, it is what we will use if we need to contact you later.
```

Reply:
I honestly just tried to rewrite this, instead of impose my own biases and opinions on the subject. I am committing this for now, but I'll have to change the phrasing later.